test/integration/login_test.go

```diff
@@ -58,20 +57,12 @@ func TestLogin(t *testing.T) {
 
 	loginOptions = newLoginOptions(server, username, password, "", true)
 
-	if err = loginOptions.GatherServerInfo(); err != nil {
+	if err := loginOptions.GatherInfo(); err != nil {
 		t.Fatalf("Error trying to determine server info: ", err)
 	}
 
-	if err = loginOptions.GatherAuthInfo(); err != nil {
-		t.Fatalf("Error trying to determine auth info: ", err)
-	}
-
-	me, err := loginOptions.Whoami()
-	if err != nil {
-		t.Errorf("unexpected error: ", err)
-	}
-	if me.Name != "anypassword:"+username {
-		t.Fatalf("Unexpected user after authentication: %v", me.Name)
+	if loginOptions.Username != "anypassword:"+username {
```

Comment:
why are we hardcoding "anypassword" here?  also I suspect the changes @liggitt is working on around user/identity mappings would affect this?